[2022-06-26 15:13:40,069] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: fraud_generation.generate_transaction scheduled__2022-06-26T15:08:38.211046+00:00 [queued]>
[2022-06-26 15:13:40,086] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: fraud_generation.generate_transaction scheduled__2022-06-26T15:08:38.211046+00:00 [queued]>
[2022-06-26 15:13:40,087] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2022-06-26 15:13:40,087] {taskinstance.py:1357} INFO - Starting attempt 1 of 2
[2022-06-26 15:13:40,087] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2022-06-26 15:13:40,114] {taskinstance.py:1377} INFO - Executing <Task(BashOperator): generate_transaction> on 2022-06-26 15:08:38.211046+00:00
[2022-06-26 15:13:40,128] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'fraud_generation', 'generate_transaction', 'scheduled__2022-06-26T15:08:38.211046+00:00', '--job-id', '198', '--raw', '--subdir', 'DAGS_FOLDER/fraud_generation_dag.py', '--cfg-path', '/tmp/tmpt0keh9ou', '--error-file', '/tmp/tmphme1dx_w']
[2022-06-26 15:13:40,130] {standard_task_runner.py:80} INFO - Job 198: Subtask generate_transaction
[2022-06-26 15:13:40,123] {standard_task_runner.py:52} INFO - Started process 169 to run task
[2022-06-26 15:13:40,242] {task_command.py:370} INFO - Running <TaskInstance: fraud_generation.generate_transaction scheduled__2022-06-26T15:08:38.211046+00:00 [running]> on host e40375faaea0
[2022-06-26 15:13:40,383] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=***@example.com
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=fraud_generation
AIRFLOW_CTX_TASK_ID=generate_transaction
AIRFLOW_CTX_EXECUTION_DATE=2022-06-26T15:08:38.211046+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-06-26T15:08:38.211046+00:00
[2022-06-26 15:13:40,385] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-06-26 15:13:40,385] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'python3 /opt/***/dags/script_generation.py 29549']
[2022-06-26 15:13:40,402] {subprocess.py:85} INFO - Output:
[2022-06-26 15:13:41,797] {subprocess.py:92} INFO - /tmp/***tmpl26jew0i
[2022-06-26 15:13:41,946] {subprocess.py:96} INFO - Command exited with return code 0
[2022-06-26 15:13:42,034] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=fraud_generation, task_id=generate_transaction, execution_date=20220626T150838, start_date=20220626T151340, end_date=20220626T151342
[2022-06-26 15:13:42,108] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-06-26 15:13:42,215] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
